topbeat/beater/topbeat.go

```diff
@@ -115,7 +115,10 @@ func (t *Topbeat) Setup(b *beat.Beat) error {
 func (t *Topbeat) Run(b *beat.Beat) error {
 	var err error
 
-	t.procStats.InitProcStats()
+	err = t.procStats.InitProcStats()
```

Comment:
@ruflin this might conflict with #1081, let's see which PR gets merged first so we can updated the other.